emva1288/process/plotting.py

```diff
@@ -168,12 +168,12 @@ def rearrange(self):
 
 
 class PlotUyDark(Emva1288Plot):
-    '''Create $\mu_{y.dark}$ plot'''
+    '''Create $\\mu_{y.dark}$ plot'''
```

Comment:
Alternatively, you could use a raw string here.
```suggestion
    r'''Create $\mu_{y.dark}$ plot'''
```

Reply:
I wasn't sure which was better at that point. Since you also suggest it, I'll take it ;-)